README.md

```diff
@@ -1464,20 +1464,25 @@ version: '2.2'
 
 services:
```

Comment:
can you align the structure of this docker-compose to the one with searchguard? elasticsearch should be the first entry, followed by kibiter and mariadb. Thanks